queue/abstract/driver/fifottl.lua

```diff
@@ -107,7 +90,7 @@ local ttl_states    = { state.READY, state.BURIED }
 local ttr_state     = { state.TAKEN }
 
 local function fifottl_fiber_iteration(self, processed)
-    local now       = time()
+    local now       = util.time()
     local task      = nil
     local estimated = TIMEOUT_INFINITY
```

Comment:
I would leave only one TIMEOUT_INFINITY constant, in the util module. This commit left TIMEOUT_INFINITY in fifottl.lua and utubettl.lua. It is inconsistent.

Reply:
OMG. Thank you. i missed it.